config/src/config/test_data/random.complete.node.config.toml

```diff
@@ -82,3 +82,7 @@ enable_remote_authentication = true
 network_peers_file = ""
 seed_peers_file = "31893204fa402143c11b26ce8a89ea1d.seed_peers.toml"
 identity_private_key = "60dd107034b4582a2ef42c5e1ea475f2fea477a10a9f1d75b3635243b2506b72"
+
+[validator_network.network_id]
+network_name = ""
```

Comment:
this shouldn't have a network_name :O
also I would just use name and type here otherwise it is a bit redundant

Reply:
`type` is a reserved word in rust :(. So, to be consistent I put `network` before both.

Reply:
Solved with a serde tag.